src/java.base/share/classes/java/io/ObjectInputFilter.java

```diff
@@ -523,10 +522,15 @@ static ObjectInputFilter rejectUndecidedClass(ObjectInputFilter filter) {
      * {@systemProperty jdk.serialFilter}, its value is used to configure the filter.
      * If the system property is not defined, and the {@link java.security.Security} property
      * {@code jdk.serialFilter} is defined then it is used to configure the filter.
-     * The filter is created as if {@link #createFilter(String) createFilter} is called;
-     * if the filter string is invalid, an {@link ExceptionInInitializerError} is thrown.
-     * Otherwise, the filter is not configured during initialization and
-     * can be set with {@link #setSerialFilter(ObjectInputFilter) Config.setSerialFilter}.
+     * The filter is created as if {@link #createFilter(String) createFilter} is called,
+     * if the filter string is invalid the initialization fails and subsequent attempts to
```

Comment:
Hello Roger,
> if the filter string is invalid the initialization fails

Should this instead be "if the filter string is invalid the initialization of {@code Config} class fails ..."

Reply:
Please ignore this above comment. Clearly, the initialization of `Config` class doesn't fail, but the initialization of the JVM wide serial filter fails. I still had the older semantics of `Config`  class in mind when I submitted that above comment.